Dockerfiles/nodejs4.3

```diff
@@ -0,0 +1,7 @@
+FROM amazonlinux
+
+RUN yum install -y wget zip
+RUN curl -sL https://nodejs.org/dist/v4.3.2/node-v4.3.2-linux-x64.tar.gz | tar zxC /usr/local --strip-components=1
+RUN rm -rf /usr/local/lib/node_modules/npm && \
+  mkdir /usr/local/lib/node_modules/npm && \
+  curl -sL https://github.com/npm/npm/archive/v2.14.12.tar.gz | tar xz -C /usr/local/lib/node_modules/npm --strip-components=1
```

Comment:
In the `6.x` image, all this npm junk is just about getting npm 5.3.0 onto the image. This means that stork will build the bundle respecting a `package-lock.json` file if the repository being bundled has one. 

I think we should also install v5.3.0 here.

Reply:
👍